pkg/controller/directvolumemigration/namespace.go

```diff
@@ -37,7 +39,7 @@ func (t *Task) ensureDestinationNamespaces() error {
 		// Create namespace on destination with same annotations
 		destNs := corev1.Namespace{
 			ObjectMeta: metav1.ObjectMeta{
-				Name:        ns,
+				Name:        nsNames[1],
```

Comment:
Seems like we should verify the length of this array before accessing numeric keys.